src/main/java/pl/edu/mimuw/DepositBankAction.java

```diff
@@ -1,9 +1,31 @@
 package pl.edu.mimuw;
 
-public abstract class DepositBankAction extends BankAction {
+import java.sql.Timestamp;
 
-  // TODO:
-  // - remove abstract modifier from this class
-  // - this class has to contain extra fields other than LoanBankAction
-  // - you have to design it's constructors/accessors/setters and implement some from BankAction
+public class DepositBankAction extends BankAction {
+  boolean isBiggerThanMillion;
```

Comment:
chyba zamiast tego lepiej metode by dodać, może to amount by sie zmienialo w BankAction?